doc/supportedRuntimeVersions.md

```diff
@@ -144,10 +145,10 @@ The information won't show in runtime image tags, but we can always check the de
 - 3.1
 
 ### php
-- 7.4, 8.1
+- 7.4, 8.1, 8.2
```

Comment:
Looks like 7.4 should be dropped now, but 8.0 should be present. Would it please be possible for you to update it?